test/e2e_node/lock_contention_test.go

```diff
@@ -0,0 +1,81 @@
+// +build linux
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2enode
+
+import (
+	"time"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const contentionLockFile = "/var/run/kubelet.lock"
+
+var _ = SIGDescribe("Lock contention [Slow] [Disruptive] [Serial] [NodeFeature:LockContention]", func() {
```

Comment:
```suggestion
var _ = SIGDescribe("Lock contention [Slow] [Disruptive] [Feature:LockContention]", func() {
```

We'll need to update the test-infra manifest. This should avoid this getting picked up by the suites that launch the kubelet without the right command line flags.

@SergeyKanzhelev do we want to consider marking this `[NodeSpecialFeature]` and skip that in the regular feature suite?

Reply:
I think this might have to stay serial? - (given that it messes with kubelet health)

Reply:
@ehashman I've made it `NodeSpecialFeature:Contention`, I think that works better than `Feature:LockContention`.

What do you think ?